test/regressions/index.tsx

```diff
@@ -1,13 +1,18 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom/client';
 import { createBrowserRouter, RouterProvider, Outlet, NavLink, useNavigate } from 'react-router';
+import { Globals } from '@react-spring/web';
 import { setupFakeClock, restoreFakeClock } from '../utils/setupFakeClock'; // eslint-disable-line
 import { generateTestLicenseKey, setupTestLicenseKey } from '../utils/testLicense'; // eslint-disable-line
 import TestViewer from './TestViewer';
 import type { Test } from './testsBySuite';
 
 setupTestLicenseKey(generateTestLicenseKey(new Date('2099-01-01')));
 
+Globals.assign({
+  skipAnimation: true,
+});
```

Comment:
One of our tests uses react spring to demonstrate how to use a custom animation library